doc/api/tls.md

```diff
@@ -1076,6 +1076,12 @@ For example:
 console.log(tls.getCiphers()); // ['AES128-SHA', 'AES256-SHA', ...]
 ```
 
+## tls.DEFAULT_ECDH_CURVE
+
+The default curve name to use for ECDH key agreement. The default value is
```

Comment:
This is only effective on the key agreement on a tls server. I think that `for ECDH key agreement in a tls server.`  is better.